README.md

```diff
@@ -1,2 +1,188 @@
-# codeowners-generator
-CODEOWNERS solution for mono repos
+<!-- PROJECT LOGO -->
+<p align="center">
+    <a href="https://www.npmjs.com/package/codeowners-generator">
+      <img src="https://img.shields.io/npm/v/codeowners-generator/latest.svg?style=flat-square" alt="NPM Version" />
+    </a>
+    <a href="https://github.com/gagoar/codeowners-generator/actions">
+      <img src="https://github.com/gagoar/codeowners-generator/workflows/codeowners-generator/badge.svg" alt="Workflow" />
+    </a>
+    <a href="https://codecov.io/gh/gagoar/codeowners-generator">
+      <img src="https://codecov.io/gh/gagoar/codeowners-generator/branch/master/graph/badge.svg?token=48gHuQl8zV" alt="codecov" />
+    </a>
+    <a href="https://github.com/gagoar/codeowners-generator/blob/master/LICENSE">
+      <img src="https://img.shields.io/npm/l/codeowners-generator.svg?style=flat-square" alt="MIT license" />
+    </a>
+</p>
+
+<p align="center">
+
+  <h3 align="center">codeowners-generator</h3>
+
+  <p align="center">
+    ✨ use codeowners anywhere in your monorepo 🛠️
+    <br />
+    <a href="https://github.com/gagoar/codeowners-generator#table-of-contents"><strong>Explore the docs »</strong></a>
+    <br />
+    <a href="https://github.com/gagoar/codeowners-generator/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/gagoar/codeowners-generator/issues">Request Feature</a>
+  </p>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [About the Project](#about-the-project)
+- [Built With](#built-with)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+[CODEOWNERS](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) are automatically requested for review when someone opens a pull request that modifies code that they own. This is a great feature, but when working on monorepos ownership is shared between teams and it becomes difficult to maintain.
+
+`codeowners-generator` allows you to position CODEOWNERS files anywhere in your project tree and it will take care of compiling all the files into a single generated file, that Github can understand. It also can read the maintainers field in `package.json` (`useMaintainers` option in the cli ) making easy to keep CODEOWNERS and package.json in sync.
+
+### Built With
+
+- [ora](https://github.com/sindresorhus/ora)
+- [commander](https://github.com/tj/commander.js/)
+- [cosmiconfig](https://github.com/davidtheclark/cosmiconfig)
+
+<!-- GETTING STARTED -->
+
+### Installation
+
+If you wish to use `codeowners-generator` as a standalone utility:
+
+```sh
+npm -g install codeowners-generator
+```
+
+This will make the `alo` command available in your terminal.
+
+```sh
+codeowners-generator --help
+```
+
+If instead you would like to add it to a package:
+
+```sh
+npm install --only=dev codeowners-generator
+```
+
+or
```

Comment:
was this meant to be here? or perhaps you're missing a section after this?

Reply:
oh yeah, I was doing yarn as well, and then I gave up.